guides/v2.3/config-guide/elasticsearch/es-config-apache.md

```diff
@@ -19,116 +19,116 @@ The reason the proxy is not secured in this example is it's easier to set up and
 
 See one of the following sections:
 
-* [Set up a proxy for Apache 2.4](#es-apache-proxy-24)
-* [Set up a proxy for Apache 2.2](#es-apache-proxy-22)
+*  [Set up a proxy for Apache 2.4](#es-apache-proxy-24)
+*  [Set up a proxy for Apache 2.2](#es-apache-proxy-22)
 
 ### Set up a proxy for Apache 2.4 {#es-apache-proxy-24}
 
 This section discusses how to configure an Elasticsearch proxy using a virtual host.
 
 1. Enable `mod_proxy` as follows:
 
-    ```bash
-    a2enmod proxy_http
-    ```
+   ```bash
+   a2enmod proxy_http
+   ```
 
-2. Use a text editor to open `/etc/apache2/sites-available/000-default.conf`
-3. Add the following directive at the top of the file:
+1. Use a text editor to open `/etc/apache2/sites-available/000-default.conf`
+1. Add the following directive at the top of the file:
 
-    ```conf
-    Listen 8080
-    ```
+   ```conf
+   Listen 8080
+   ```
 
-4. Add the following at the bottom of the file:
+1. Add the following at the bottom of the file:
 
-    ```conf
-    <VirtualHost *:8080>
-        ProxyPass "/" "http://localhost:9200/"
-        ProxyPassReverse "/" "http://localhost:9200/"
-    </VirtualHost>
-    ```
+   ```conf
+   <VirtualHost *:8080>
+       ProxyPass "/" "http://localhost:9200/"
+       ProxyPassReverse "/" "http://localhost:9200/"
+   </VirtualHost>
+   ```
 
-5. Restart Apache:
+1. Restart Apache:
 
-    ```bash
-    service apache2 restart
-    ```
+   ```bash
+   service apache2 restart
+   ```
 
-6. Verify the proxy works by entering the following command:
+1. Verify the proxy works by entering the following command:
 
-    ```bash
-    curl -i http://localhost:<proxy port>/_cluster/health
-    ```
+   ```bash
+   curl -i http://localhost:<proxy port>/_cluster/health
+   ```
 
-    For example, if your proxy uses port 8080:
+   For example, if your proxy uses port 8080:
 
-    ```bash
-    curl -i http://localhost:8080/_cluster/health
-    ```
+   ```bash
+   curl -i http://localhost:8080/_cluster/health
+   ```
 
-    ```terminal
-    Messages similar to the following display to indicate success:
+   ```terminal
+   Messages similar to the following display to indicate success:
```

Comment:
This line should be outside the code block. (See line 121 below.)

Reply:
Thanks @keharper. Done.